.gitignore

```diff
@@ -6,4 +6,5 @@ node_modules
 docs
 coverage
 /build
+/simulations/build
```

Comment:
What do "simulations" mean?

Reply:
Simulations are web pages that run parts of the extension panel from within a normal web page to simulate how those parts behave in the extension panel with some set of devtool protocol input.